### DISCUSSION TITLE
CPU limiter

### DISCUSSION THREAD

**Discussion by johnlrhodes757** [revoxhere/duino-coin#743] [Ideas]
So I like to use my CPU to mine but at the same time have it set to 1% at the moment.  At 1% it is hashing at 3.22 MH/s per core.  Is there any way to introduce a limit into the PC miner where it only allows for lets say 200 H/s???  I have under clocked my PC also and even took it down from 16 threads to 8 threads.  I wish there was a way to take it lower then 1%.

**Comment by revoxhere** [comment]
No

**Reply by johnlrhodes757** [reply]
Well I guess I will just desert the CPU miner for now then.  Way to strong for my comfort on the network.  I'll  look into trying to figure out if windows has a way to limit processes and try it again later.  Thanks revox

**Reply by primitt** [reply]
What..... there is no point in that, 1 percent is the efficiency and wont change the hashrate

**Reply by johnlrhodes757** [reply]
when i set it to 95% it hashes at 6.7 MH/s

**Reply by marcoduinocoin** [reply]
> What..... there is no point in that, 1 percent is the efficiency and wont change the hashrate

Totally agreed.
